rust/agents/relayer/src/msg/serial_submitter.rs

```diff
@@ -228,7 +228,7 @@ impl SerialSubmitter {
                 info!(msg=?msg, "Message processed");
             }
             Err(e) => {
-                info!(msg=?msg, "Message processing failed: {}", e);
```

Comment:
Please, don't put formatted data into the structured message. It makes it very hard to filter. It is much better if it is added as a separate field to the object.

Reply:
(I realize it was that way before, but great time to fix)

Reply:
Agreed and moved it out